README.md

```diff
@@ -5,14 +5,14 @@ that puma can provide:
 
 Gauges:
 
-* puma.workers - The number of distinct process running. In single mode this will be 1, in cluster mode the number of worker processes
+* puma.workers - The number of distinct processes running. In single mode this will be 1, in cluster mode the number of worker processes
 * puma.old_workers - The number of worker processes that are about to be shut down as part of a phased restart. Will normally be 0
 * puma.booted_workers - The number of worker processes that are in a booted state
-* puma.running - The number of worker threads currently running. In quiet times, idle threads may be shutdown, but they'll start back up again when traffic arrives
-* puma.backlog - The number of requests that have made it to a worker but are yet to be processed. This will normally be zero, as requests queue on the tcp/unix socket in front of the master puma process, not in the worker thread pool
-* puma.pool_capacity - The number of idle and unused worker threads. When this is low/zero, puma is running at full capacity and might need scaling up
+* puma.running - The number of worker threads currently running. In quiet times, idle threads may be shutdown if min threads is less than max threads, but they'll start back up again when traffic arrives
+* puma.backlog - The number of requests that have made it to a worker but are yet to be processed. This will normally be zero if queue_requests is disabled, as requests will only queue on the tcp/unix socket in front of the master puma process, but not in the worker thread pool
```

Comment:
https://github.com/puma/puma/blob/13e18e8078c800adfc52af687acc1d8de5f3988d/lib/puma/dsl.rb#L699-L715

Reply:
interesting. `queue_requests` is enabled by default though, so does that mean `This will normally be zero` is typically not true?

It's been a while since I worked with these stats closely, but my recollection is that this value was nearly always `0` in my apps that used cluster mode 🤔

Reply:
> so does that mean This will normally be zero is typically not true?

Ah yep that wording doesn't make sense anymore.

> but my recollection is that this value was nearly always 0 in my apps that used cluster mode 🤔

When `queue_requests` is enabled, each worker has a reactor thread that is responsible for queuing and read-buffering requests. However, and this was surprising to me as well, when there's a new connection, worker threads try to eagerly read the request off the socket first, and only hand it over to the reactor if it's not ready. I'm [trying](https://github.com/puma/puma/issues/3501) to change that behavior, but that might explain why your backlog is close to 0, it could just be that your requests are always processed before they can get queued.